scripts/zkai_native_attention_mlp_rmsnorm_label_sensitivity_gate.py

```
@@ -0,0 +1,661 @@
+        "non_claims": list(NON_CLAIMS),
+        "validation_commands": list(VALIDATION_COMMANDS),
+    }
+    refresh_payload_commitment(payload)
+    if include_mutations:
+        payload["mutation_result"] = run_mutations(payload)
+        refresh_payload_commitment(payload)
+    validate_payload(payload)
+    return payload
+
+
+def validate_payload(payload: dict[str, Any]) -> None:
```

Ruff (0.15.12)

[warning] Too many branches (28 > 12)

(PLR0912)